cdap-docs/cdap-apps/source/tracker/index.rst

```diff
@@ -0,0 +1,441 @@
+information and links for each.
+
+**On the left side** is the **Filter** pane, which provides information on what was found (the
+entities and metadata types) with statistics of the number found for each category. A blue
+checkbox allows you to filter based on these attributes. If you mouse over a category, an
+``only`` link will appear, which allows you to select *only* that category as a filter.
+
+Note that the *entities* and *metadata* filters have an ``and`` relationship; at least one
+selection must be made in each of *entities* and *metadata* for there to be any results
+that appear.
+
+**On the right side** is a sortable list of results. It is sortable by *Create Date* or the entity
+ID (name), either *A-Z* (alphabetical ascending), or *Z-A* (alphabetical descending).
+
+Each entry in the list provides a summery of information about the entity, and its name is
+a hyperlink to further details: metadata, lineage, and audit log.
+
+The **Jump** button provides three actions: go to the selected entity in CDAP, or add it
+to a new Cask Hydrator pipeline as a source or as a sink. Datasets can be added as sources or
+sinks to batch pipelines, while streams can be sources in batch pipelines or sinks in
+real-time pipelines.
+
+Entity Details
+--------------
+Clicking on a name in the search results list will take you to details for a particular
+entity. Details are provided on three separate tabs: *Metadata*, *Lineage*, and *Audit
+Log*.
+
+**Metadata**
+
+The *Metadata* tab provides lists of *System Tags*, *Schema*, *User Properties*, and
+*System Properties* that were found for the entity. The values shown will vary depending
+on the type of entity and each individual entity. For instance, a stream may have a schema
+attached, and if so, it will be displayed.
+
+.. figure:: ../_images/tracker-metadata.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+**Lineage**
+
+The *Lineage* tab shows the relationship between an entity and the programs that are
+interacting with it. As different lineage diagrams can be created for the same entity,
+depending on the particular set of programs selected to construct the diagram, a green
+button in the shape of an arrow is used to cycle through the different lineage digrams
+that a particular entity participates in.
+
+A date menu in the left side of the digram lets you control the time range that the
+diagram displays. By default, the last seven days are used, though a custom range can be
+specified, in addition to common time ranges (two weeks to one year).
+
+.. figure:: ../_images/tracker-lineage.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+**Audit Log**
+
+The *Audit Log* tab shows each record in the *AuditLog* dataset that has been created for
+that particular entity, displayed in reverse chronological order. Note that due to a
+limitation in CDAP, reading and writing from a flow or service to a dataset shows an
```

Comment:
instead of a limitation, should we say something like "Because of how datasets work in CDAP, reading and writing..." Additional visibility will be available in future releases?